### DISCUSSION TITLE
[question] How add meta tag in index page?

### DISCUSSION THREAD

**Discussion by hezhangjian** [facebook/docusaurus#8857] [General]
Hello, community. I want to make my github pages paas the verified domain property owner in [Google Search Console](https://search.google.com/search-console/about). I would like to be verified by HTML tag. Which needs add a HTML in website index page. 

Like youtube https://stackoverflow.com/questions/57384269/github-pages-blog-and-google-search-console-is-it-safe-to-follow-these-steps-fo
```
TXT | youtube.com | google-site-verification=OQz60vR-YapmaVrafWCALpPyA8eKJKssRhfIrzM-DJI
```

How could I achieve that? Thanks for your help.

**Comment by slorber** [comment]
> TXT | youtube.com | google-site-verification=OQz60vR-YapmaVrafWCALpPyA8eKJKssRhfIrzM-DJI


This is a DNS record and not a page metadata. You should add this to your DNS settings instead.

You can still add metadata to your index page or all your site page with multiple solutions:
- https://docusaurus.io/docs/seo
- https://docusaurus.io/docs/markdown-features/head-metadata
